Documentation/ExtensionArchitecture/FileStructure/Configuration/TCA/Index.rst

```diff
@@ -14,6 +14,14 @@ The folder :file:`EXT:my_extension/Configuration/TCA/` may contain or override
 All files in this directory are automatically included during the TYPO3
 :ref:`bootstrap <bootstrapping>`.
 
+..  versionadded:: 12.0
+    Files within :file:`Configuration/TCA/` files are loaded within a dedicated scope.
+    This means that variables defined in those files can't leak into the following files.
```

Comment:
Does an average user understand this? "leak into the following files". What does that mean? A user may only have one TCA/Overrides files, so there may be no following files.

Reply:
Sounds better for me.